src/main/java/com/sap/sgs/phosphor/fosstars/data/github/UsesGithubForDevelopment.java

```diff
@@ -29,31 +29,19 @@
    */
   private static final List<Predicate<GHRepository>> CHECKS = Arrays.asList(
 
-      // check if the repository doesn't have an explicit link to the original repository
-      repository -> StringUtils.isEmpty(repository.getMirrorUrl()),
-
       // check if the description mentions "mirror"
       repository -> !repository.getDescription().toLowerCase().contains("mirror"),
 
-      // if GitHub issues are enabled, then it's likely that the project uses GitHub
-      GHRepository::hasIssues,
```

Comment:
Why do you drop this check?

Reply:
Well with `hasIssues()`, the output gave more False Positives
Like all `github/eclipse` projects were giving output as `true`. Which is a False Positive. Like https://git.eclipse.org/c/?q=jetty.project.

Reply:
I think Jetty Project does use GitHub for development:

- https://www.eclipse.org/jetty/documentation/current/contributing-patches.html mentions only GitHub
- They take advantage of many GitHub feature such as issue templates https://github.com/eclipse/jetty.project/tree/jetty-10.0.x/.github/ISSUE_TEMPLATE

Reply:
You are right about this https://github.com/eclipse/jetty.project/blob/jetty-10.0.x/CONTRIBUTING.md#developer-resources confirms that the jetty.project uses GitHub.

Reply:
This check is re-added.